modules/ROOT/pages/log-points.adoc

```diff
@@ -1,17 +1,137 @@
 = Anypoint Monitoring Log Points
 
-The Anypoint Monitoring log points feature enables you to interactively extract data from running managed APIs. You can specify properties (for example, the host and method in the request header) in your managed API for which to generate logs without having to write code or redeploy your application. This helps you troubleshoot without having to restart the application. 
+== Log Points for Apps
+
+The Log Points feature for apps enables you to log information from connectors in any Mule app in real time. If you are using one of the supported connectors, you can select a log level (debug, info, warn, or error) and generate logs.
+
+=== Prerequisites
+
```

Comment:
Notice that the Prereq text below includes a transition fragment that leads to the list.
Choose one format and use it in both (all) peer texts consistently.
I do not feel strongly about which format you use.

Reply:
```suggestion

To use this feature, you must have the following:

```